src/main/java/hudson/plugins/logparser/LogParserPublisher.java

```diff
@@ -206,9 +240,36 @@ public boolean configure(final StaplerRequest req, final JSONObject json)
                     "log-parser.").toArray(new ParserRuleFile[0]);
             useLegacyFormatting = json.getJSONObject("log-parser").getBoolean(
                     "useLegacyFormatting");
+            JSONObject colors = json.optJSONObject("log-parser");
+            if (colors != null) {
+                colorWarning = colors.optString("colorWarning");
```

Comment:
What do you mean? If a user is specifying a string which is not a color, like "blah"? The thing is how do you check the string is a valid css color without a giant array of all the valid colors? I'm already doing form validation on hex color and string (which should be letters only words), but I don't know how to test that.